src/reviewer.py

```diff
@@ -249,6 +250,8 @@ def remove_hooks(self):
             hooks.card_did_leech.remove(self.save_leech)
         except NameError:
             print(ErrorMsg.ACTION_MANAGER_NOT_DEFINED)
+        except AttributeError:
```

Comment:
for some reason, sometimes the hooks.card_did_leech does not have anything, in this case we do nothing. Hopefully it won't have any negative effects